src/chart/rendering/painter/coordinatesystem.cpp

```diff
@@ -21,7 +21,7 @@ Point PolarDescartesTransform::convert(const Point &p) const
 
 	auto mapped = mappedSize();
 	auto usedAngle =
-	    Math::interpolate(0.0, 2.0 * M_PI, (double)polar);
+	    Math::interpolate(0.0, 2.0 * M_PI, static_cast<double>(polar));
```

Comment:
the protected polar member could be double, then we would not need these casts.

Reply:
I handle it in a separated PR

Reply:
`polar` means "isPolar" as type FuzzyBool. I think this conversion is right. (If I rewrite to double, other casts needed because fuzzybool boolean logic used with this variable)